Assignment5Objects/src/assignment5objects/MainProgram.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * @author Samantha Shurie
+ * Assignment 5: Working with Objects
+ */
+package assignment5objects;
+
+
+public class MainProgram {
+   
+    /**
+     * Main method implements both the RetailItem and Car classes. This method
+     * creates a new instance of the Car class and increases/decreases it's speed
+     * field. It also creates three instances of the RetailItem class and prints
+     * their fields.
+     * @param args 
+     */
+    public static void main(String[] args) {
+        int currentSpeed = 0;   //used to retrieve speed value of car object
+         
+        println("EXERCISE 2: CAR CLASS\n");
+        
+        Car car = new Car(2010, "Honda");
+
+        println("\tAccelerating...");
+
+        for(int i = 0; i < 5; i++) {
+            car.accelerate();
+            currentSpeed = car.getSpeed();
```

Comment:
I understand why you did this but for something this simple, placing the work inside the print statement will be just as easy to read. It also saves you from doing an extra step and needing to create a variable to do it.